src/scripts/main.js

```diff
@@ -1 +1,58 @@
+/* eslint-disable prefer-promise-reject-errors */
 'use strict';
+
+function createMessage(message, className) {
+  document.body.insertAdjacentHTML(
+    'afterbegin',
+    `
+    <div data-qa="notification" class="${className}">
+    ${message}
+    </div>
+    `,
+  );
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve();
+  });
+
+  setTimeout(() => {
+    reject();
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const thirdPromise = new Promise((resolve) => {
+  let leftCounter = 0;
+  let rightCounter = 0;
+
+  document.addEventListener('click', () => {
+    leftCounter++;
+
+    if (leftCounter > 0 && rightCounter > 0) {
+      resolve();
+    }
+  });
+
+  document.addEventListener('contextmenu', () => {
+    rightCounter++;
+
+    if (leftCounter > 0 && rightCounter > 0) {
+      resolve();
+    }
+  });
+});
+
+firstPromise.then(() => createMessage('First promise was resolved', 'success'));
+firstPromise.catch(() => createMessage('First promise was rejected', 'error'));
+
+secondPromise.then(() =>
+  // eslint-disable-next-line prettier/prettier
```

Comment:
Let's try NOT to disable linter rules. Isn't there a better way?

Reply:
Had to try it locally, really, for some reason linter has a strange behaviour here. It works as it should to, only if we add .catch chaining, but it's not what we want here, so let's leave it as it is.